diff --git a/dist/streaming/createReactStreamingHandler.js b/dist/streaming/createReactStreamingHandler.js
index 4dc6546114ad3985f1c922f607f95421478c7940..561991625a7d52d7072a0aaaabf583dbdf0502bc 100644
--- a/dist/streaming/createReactStreamingHandler.js
+++ b/dist/streaming/createReactStreamingHandler.js
@@ -76,7 +76,7 @@ const createReactStreamingHandler = async ({
       jsBundles,
       res
     }, {
-      waitForAllReady: isSeoCrawler
+      waitForAllReady: false
     });
   };
 };
